game/scripts/vscripts/components/duels/duels.lua

```diff
@@ -47,6 +47,12 @@ function Duels:CheckDuelStatus (keys)
   if not Duels.currentDuel or Duels.currentDuel == DUEL_IS_STARTING or keys.killed:IsReincarnating() then
     return
   end
+  if keys.killed:IsReincarnating() then
+    keys.killed:SetRespawnsDisabled(false)
+  end
+  Timers:CreateTimer(1, function ()
+    keys.killed:SetRespawnsDisabled(true)
```

Comment:
this code will also never run since there's a check to `or keys.killed:IsReincarnating() then` right before it